itou/templates/static/legal/terms.html

```diff
@@ -22,274 +18,754 @@ <h2 class="h3">
                     <p>
-                        « L’Employeur inclusif » est la personne travaillant pour le compte de la structure professionnelle proposant des offres d’emploi pour des candidats éligibles par le biais des Services offerts par les emplois de l&#39;inclusion visant à faciliter l’insertion professionnelle au travers de structures de l’inclusion.
+                        « Le Prescripteur habilité »  est tout organisme ou personne habilité à évaluer, à l'issue d'un
+                        diagnostic socioprofessionnel, l'éligibilité d'un candidat au dispositif IAE
+                        conformément aux dispositions législatives et réglementaires en vigueur.
                     </p>
+                    <p>
+                        « L'Orienteur »  est un organisme ou une personne pouvant orienter et accompagner dans ses démarches un
+                        candidat. Il n'est pas chargé d'évaluer et valider à l'issue
+                        d'un diagnostic socioprofessionnel, l'éligibilité à l'IAE. Il est un prescripteur qui ne fait pas partie d'une organisation habilitée à
+                        évaluer, à l'issue d'un diagnostic socioprofessionnel,
+                        l'éligibilité d'un candidat au dispositif IAE.
 
-                    <h2 class="mt-3 mt-lg-5" id="article-4--fonctionnalités">Article 4- Fonctionnalités</h2>
-                    <h3 id="41-création-de-lespace-personnel">4.1 Création de l’Espace personnel</h3>
+                    </p>
+                    <p>
+                        « L'Employeur »  est la personne travaillant pour le compte de la structure
+                        professionnelle proposant des offres d'emploi pour des candidats éligibles par le biais des
+                        Services offerts par les Emplois de l'inclusion visant à faciliter l'insertion
+                        professionnelle au travers de structures de l'inclusion. Il s'agit des Structures
+                        d'insertion par l'activité économique, des Entreprises adaptées, des
+                        Entreprises adaptées de travail temporaire, des Groupements d'employeurs pour
+                        l'insertion et la qualification, ainsi que des Organismes porteurs de la clause sociale.
+                    </p>
+                    <p>
+                        « Les Institutions partenaires »  sont les organismes publics ou chargés par ces
+                        derniers, habilités à piloter et contrôler les éléments
+                        d'éligibilité aux dispositifs d'insertion conformément aux textes
+                        législatifs et réglementaires en vigueur.
+                    </p>
+                    <p>
+                        « Le PASS IAE »  valide l'entrée à un parcours d'insertion pour
+                        l'activité économique pour une durée de 2 ans. Il permet au candidat d'être recruté par toutes les <a class="has-external-link"
+    target="_blank"
```

Comment:
Est-ce qu'on veut pas aussi rajouter ?

```suggestion
    target="_blank"
    rel="noreferrer"
```